dbux-data/src/callGraph/CallGraph.js

```diff
@@ -262,14 +262,16 @@ export default class CallGraph {
     const trace = this.dp.collections.traces.getById(traceId);
     const realContextId = this.dp.util.getRealContextIdOfTrace(traceId);
     const parentTraces = this.dp.indexes.traces.parentsByRealContext.get(realContextId) || EmptyArray;
+    const callerTraces = parentTraces.map(t => this.dp.util.getPreviousCallerTraceOfTrace(t.traceId))
```

Comment:
Why do we have both, `parentTraces` + `callerTraces`?

Reply:
Yes this should be fixed. Previously, `ExecutionContext.parentTraceId` might be an `Argument` trace rather than a `BCE` trace, so `dp.util.getPreviousCallerTraceOfTrace` is used to find the actuall BCE (in similar function `_getNextChildTrace` so I copied the code from that). But now `parentTraceId` is the actual BCE if the context is created by a `CE`, so we may remove the function here.